### PR TITLE
Improve AC ORM measurement class

### PR DESCRIPTION
**Pull request by fernandohds564**
This PR brings several changes to the class, including:
 - A new way of measuring RF line using the conditioning mode of the LLRF system;
 - Adaptation of the measurement methods to the new interfaces of the acquisition classes;
 - Addition of scaling factors on the analysis to match STD of matrix columns with the ones from the reference matrix;
 - Improvements in the algorithm for phase detection. This is important to get the correct signal of the matrix elements;
 - Methods to check sanity of measurement and results, which will allow automatic looping through several measurements;
 - Addition of several methods to plot results and check robustness of analysis;
 - Control of logging during measurement and analysis.

There is an updated notebook in `screens_iocs/2023-10-13_SI_AC_ORM_tests` exemplifying how to measure and process the data and use the plotting methods.

### CODE REVIEW

File: apsuite/commisslib/meas_ac_orm.py [hunk 1]
```diff
@@ -2,80 +2,121 @@
...
-        stg += ftmp('rf_kick', self.rf_kick, '[Hz]')
-        stg += ftmp('delay_corrs', self.delay_corrs, '[s]')
-        stg += ftmp('delay_rf', self.delay_rf, '[s]')
-        stg += ftmp('exc_duration', self.exc_duration, '[s]')
-        stg += ftmp('exc_rf', self.exc_rf, '[s]')
-        stg += stmp('ch_freqs', str(self.ch_freqs), '[Hz]')
-        stg += stmp('cv_freqs', str(self.cv_freqs), '[Hz]')
-        stg += dtmp('nr_sectors_per_acq', self.nr_sectors_per_acq, '')
-        stg += stmp('sectors_to_measure', str(self.sectors_to_measure), '')
+        stf += stmp('ref_respmat_name', self.ref_respmat_name, '')
```
Comment (vellosok75) [inline apsuite/commisslib/meas_ac_orm.py:70]: `stf` -> stg

**Review by anacso17 — APPROVED** [review]

**Review by vellosok75 — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Comment by fernandohds564** [comment]
Guys, I asked your review again because a lot has changed since you looked at this PR.

File: apsuite/commisslib/meas_ac_orm.py [hunk 2]
```diff
@@ -401,340 +552,922 @@ def get_general_data(self):
...
+        Args:
+            mat_ac (numpy.array, (320, 281), optional): The measured matrix.
+                Defaults to None.
+            mat_dc (numpy.array, (320, 281), optional): The reference matrix.
+                Defaults to None.
+
+        Returns:
+            matplotlib.Figure: Figure object of the plot.
+            tuple: Tuple with both axes of the figure;
+            col_dcorr (np.ndarray, (2, 281)): One minus colunm space
```
Comment (murilobalves) [inline apsuite/commisslib/meas_ac_orm.py:624]: typo "colunm"

Comment (fernandohds564) [reply apsuite/commisslib/meas_ac_orm.py]: thanks, done!

**Review by murilobalves — APPROVED** [review]

**Review by Gabrielrezende-asc — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Review by vellosok75 — APPROVED** [review]

**Pull request merged**